### PR TITLE
[#34] Use new raptors logo

### PR DESCRIPTION
**Pull request by kammanz**
Pretty straight forward big fix. In order to match correct design size, I had to use height 40px, even though sketch file says 32px.

### CODE REVIEW

File: src/scss/App.scss [hunk 1]
```diff
@@ -7,6 +7,25 @@ $border-dark-grey: #919191;
...
+
+//Player Card Number
+$grey3: #959595;
+
+// Player Card Position
+$grey4: #777777;
+
+// Header Links, Player Card Name
+$grey5: #4e4e4e;
+
```
Comment (Tumne) [inline src/scss/App.scss:29]: I see why you put the comments here, though, once we refactor the scss, we shouldn't put comments. We'll also figure out a better naming system for our colors. Naming color variables is very open to interpretation, but we'll find something better than `grey5`

If you could add a TODO comment above this section and we'll get to it when we get to it:

`// TODO: rename color variables and remove temporary comments`

File: src/scss/App.scss [hunk 2]
```diff
@@ -36,11 +55,14 @@ body {
...
+///// Header Styling /////
+
 .header-container {
+  position: relative;
   display: flex;
-  height: 45px;
   justify-content: space-evenly;
-  box-shadow: 1px -1px 2px $border-shadow;
+  height: 45px;
+  box-shadow: -1px -3px 10px $grey2;
```
Comment (Tumne) [inline src/scss/App.scss:66]: 👍  Good eye!

File: src/scss/App.scss [hunk 3]
```diff
@@ -52,22 +74,23 @@ body {
...
-      padding: 0 15px;
+      padding: 0 15px 0 0;
       height: 100%;
       border: none;
       background: none;
       color: $white;
       font-family: spurs;
       font-size: 22px;
-      letter-spacing: 1px;
+      letter-spacing: 2px;
```
Comment (Tumne) [inline src/scss/App.scss]: I prefer `letter-spacing: 1px`

File: src/scss/App.scss [hunk 4]
```diff
@@ -52,22 +74,23 @@ body {
...
     .logo-container {
-      display: flex;
-      border-radius: 24px;
-      box-shadow: 1px 1px 5px $border-dark-grey;
-      margin: 0 15px;
-      background-color: white;
+
+      img {
+        display: block;
+        margin: 0 10px;
```
Comment (Tumne) [inline src/scss/App.scss]: Let's make this `margin-left: 10px;`

File: src/scss/App.scss [hunk 5]
```diff
@@ -52,22 +74,23 @@ body {
...
+        height: 40px;
+      }
     }
 
     button {
-      padding: 0 15px;
+      padding: 0 15px 0 0;
```
Comment (Tumne) [inline src/scss/App.scss]: Let's leave this as `padding: 0 15px;`
and make the img only have `margin-left: 10px;` (see comment above)

File: src/scss/App.scss [hunk 6]
```diff
@@ -173,7 +196,199 @@ body {
     border-right: 6px solid transparent;
     margin: 15px;
   }
+} 
+
+///// Search Bar Styles /////
```
Comment (Tumne) [inline src/scss/App.scss]: From `Line 201` and below, the scss shouldn't be included in this PR. 
It's possible you didn't branch off of `master` but rather your other open PR.

Eitherway, Let's remove all the scss from `Line 201` and below, as this code has nothing to do with this task.

File: src/scss/App.scss [hunk 7]
```diff
@@ -36,11 +55,14 @@ body {
   justify-content: space-evenly;
 }
 
+///// Header Styling /////
```
Comment (Tumne) [inline src/scss/App.scss]: FYI, the preferred way to label sub-categories is
`// Header Styling`

**Review by Tumne — CHANGES_REQUESTED** [review]
The new logo looks legit. A couple of minor adjustments and we should be g2g.

**Comment by kammanz** [comment]
Fixed changes and ready for next PR review.

File: src/scss/App.scss (hunk 6 above)
Comment (kammanz) [reply src/scss/App.scss]: I thought I had branched off of master but I will double check. As an aside, I had installed that program you recommended, sourcetree, but it was giving issues after setup and I just wanted to get on with my work. 

EDIT: used git log --graph and it looks like it was branched off of master if i'm reading this correctly.

Comment (Tumne) [reply src/scss/App.scss]: Well, that scss code wasn't from master. I believe what had happened is you saved changes for the scss from another branch (i.e `RAP-27c`) into this branch. Just be mindful of that, no bigs

File: src/components/Header.js [hunk 8]
```diff
@@ -10,7 +11,7 @@ const Header = () => {
         <div className="header-container">
             <div className="team">
                 <div className="logo-container">
-                    <img src="https://cdn.nba.net/assets/logos/teams/secondary/web/TOR.svg" height="28" title="raptors logo" alt="raptors logo"></img>
+                    <img src={raptorsLogo} title="raptors logo" alt="raptors logo"></img>
```
Comment (Tumne) [inline src/components/Header.js:14]: Okay, so this entire section should have the `` element inside of the `<button>` one. Let's change the code for this jsx as well as the relative scss.

Firstly, Let's restructure this html to look like so:

```
        <div className="header-container">
            <div className="team">
                <button>
                    <img src={raptorsLogo} title="raptors logo" alt="raptors logo"/>
                    <span>Toronto Raptors</span>
                    <i className="drop-down-team"/>
                </button>
            </div>

            ...
```

NOTE: When you have an empty element like `<i className="drop-down-team"></i>` it should always be written as `<i className="drop-down-team"/>`

And then in App.scss replace all the `.team` css with the code below:

```
  .team {
    position: relative;
    display: flex;
    background: $background-color-purple;

    button {
      display: flex;
      align-items: center;
      padding: 0 15px 0 10px;
      height: 100%;
      border: none;
      background: none;
      color: $white;
      font-family: spurs;
      font-size: 22px;
      letter-spacing: 1px;
      text-transform: uppercase;
      cursor: pointer;

      img {
        height: 35px;
      }

      span {
        margin: 2px 13px 0 8px;
      }
    }

    &::after {
      content: '';
      position: absolute;
      left: 100%;
      z-index: 1;
      border-top: 45px solid $background-color-purple;
      border-right: 16px solid transparent;
    }
  }
```

ANOTHER NOTE: As you've noticed, the sketch designs don't always translate the correct dimensions for items. In this case, I've had to really eyeball and tweak pixel by pixel till it looks correct on the screen. The lesson here being, that a front end dev should alway have a designers mindset.

**Review by Tumne — CHANGES_REQUESTED** [review]
Looks good, though just realized the raptors logo should be within the button element. Also while we're here the angle on the angled div ends is a little off from the design.

These ones:

Screen Shot 2020-02-19 at 11 32 53 AM 
      Screen Shot 2020-02-19 at 11 32 58 AM

Let's slightly change the angles, from 

`border-right: 20px solid transparent;` 

to 

`border-right: 16px solid transparent;` 

for both `&::after` code sections.

File: src/components/Header.js (hunk 8 above)
Comment (kammanz) [reply src/components/Header.js:14]: Cool. This is different from what I've heard at the other place I worked. Their view was the dev is on a tight timeline and beholden to the client, and that designers will send you stuff but ultimately it's up to you to judge how long it will take and whether a particular design element is worth the time. But your point is taken, and that technique you showed me yesterday mapping the sketch file to the browser window will help.

Comment (Tumne) [reply src/components/Header.js:14]: The other place you worked at was wrong, lol. There is always a battle between timeline and accuracy no doubt, but a good front end dev will be able to do both. Not that I don't appreciate the point you brought up. I just wish your last job spent more time on accuracy and scalability than on renaming Chinese girls and smoking weed binge watching mediocre sci fi comedy series. 💯

**Review by Tumne — APPROVED** [review]
🏀
